### DISCUSSION TITLE
Add support for adding tooltip to Tree title columns

### DISCUSSION THREAD

**Discussion by jinsediaoying** [godotengine/godot-proposals#8332] [GUI]
add method:
Tree.SetColumnTooltip (columnIndex, tooltip);

**Comment by jinsediaoying** [comment]
#8331

**Comment by theraot** [comment]
I don't see you adding anything new here.

Perhaps you think you are getting more views, but that is unlikely, you are reaching the same set of people.

I suggest patience. People have different schedules and live in different time zones.

*I upvoted your other post, I'm not somebody you reached because of this new one. I'll go post there now, as I don't think the conversation should continue here.*

**Reply by jinsediaoying** [reply]
This post is about tooltip, and the other post is about icon.
I opened a new post because I think they are two different requests.
But as you suggest, I'll merge them into one post.

**Comment by AThousandShips** [comment]
Superseded by https://github.com/godotengine/godot-proposals/discussions/8331
